examples/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
+add_subdirectory(common)
```

Comment:
Nit: we can also add ` include_directories(common)`, and remove the dependencies from subfolders ?

Reply:
good one, thanks